### ISSUE TITLE
Include output tables into the history

### ISSUE THREAD

**Issue by lecorguille** [PF2-pasteur-fr/SARTools-Galaxy#5]
Since the tables are only available through a HTML page, it doesn't allow to use those outputs as inputs for some further steps (and within a workflow) like Filter or Graphical tools.
It's a dead-end.

Dataset collection or not Dataset collection? Good question!

Indeed, there are already a bunch of outputs. But I believe that some can be avoid.
 - table (HTML) : removed because the tables should be included into the history
 - log (text) : redirected in the stdout because the report is aleady well instructive.
 - figures : removed because there are already shipped with the report
 - RData : should be optional?

**Comment by hvaret** [comment]
I agree with you, we could simplify the output of SARTools in Galaxy:

- if it is possible, retrieve the tables (up, down and complete) to make them available within the history, and then delete the "tables" directory
- delete the figures directory
- the RData file is useful only for R users but in this case they should be able to use SARTools with R directly
- the log file contains important messages generated by SARTools/R but I agree that it could be redirected to the stdout
